src/main/java/seedu/duke/commands/DeleteCommand.java

```diff
@@ -61,10 +61,27 @@ public void execute(TextUi ui, ListManager listManager) {
             String title = information.trim();
             deleteBookmark(bookList, bookmarks, title, ui);
             break;
+        case TAG_QUOTE:
+            QuoteList quotes = (QuoteList) listManager.getList(ListManager.QUOTE_LIST);
+            deleteQuote(quotes, ui, information);
+            break;
```

Comment:
Good work. I think `information` as a parameter of this method is redundant as it can be accessed across the Class `DeleteCommand` though